src/utils/useDimensions.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react';
+import { Dimensions, ScaledSize } from 'react-native';
+
+export function useDimensions(): { window: ScaledSize; screen: ScaledSize } {
```

Comment:
Let's use the same arrow function from others utils

```suggestion
export const useDimensions = (): { window: ScaledSize; screen: ScaledSize } => {
```